### ISSUE TITLE
completion-stats-data size concern

### ISSUE THREAD

**Issue by sabi0** [JetBrains/intellij-stats-collector#17]
I was surprised to discover config/plugins/completion-stats-data folder occupies ~500 MB.

First of all, it shouldn't be there. And I welcome the switch to system/completion-stats-data.
Unfortunately the old data in config/plugins/completion-stats-data was left behind. I believe it should have been moved to system/completion-stats-data or deleted automatically.

Secondly, I see no reason for completion stats to be stored in plain text. At the very least they could be compressed.
This might not only bring the disk use improvement but also have a positive effect on performance. See for instance this discussion about compressing IDEA's caches: https://intellij-support.jetbrains.com/hc/en-us/community/posts/206932955-IDEA-cache-size

**Comment by bibaev** [comment]
@sabi0 Thanks for your report. This is a bug. We limit the size of logs at 2mb but in your case old logs not deleted for some reason. Was the internet connection established when you discovered this issue?

**Comment by sabi0** [comment]
I have a permanent connection here. But it requires a proxy to go to internet. Could that be the reason?
The proxy is configured in IDEA settings. So I have no problem for example to browse plugins repository.

P.S. The new system\completion-stats-data is already 70 MB with 309 files going back to April 24th.

**Comment by bibaev** [comment]
Yes, it is. Currently, the plugin does not use the configured proxy for data sending. Also, we should clear logs storage on write operation instead of before sending. I'll fix this in the next version.

**Comment by sabi0** [comment]
Thanks.

This is actually the second time I ran into "plugin does not use proxy" problem.
Do you think it would make sense to implement corresponding inspection for plugin developers?
(I cannot find other IDEA plugin specific inspections though..)

**Comment by bibaev** [comment]
I've checked one more time. And the plugin uses a proxy. There is no need to make such inspection because IDEA uses a global proxy selector. 

So, the issue is not in the proxy usage. Are you sure your proxy configured properly?

**Comment by sabi0** [comment]
Yes, I've just installed a new plugin from repository today.
I suppose IDEA would not be able to download it if proxy wasn't setup correctly.

I have Completion Stats Collector 0.1.2

P.S. Here is the other proxy issue in a plugin I mentioned above: https://github.com/jansorg/BashSupport/issues/526
